g/gonum.org_v1_gonum/gonum.org_v1_gonum_ubi_8.5.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : gonum
+# Version       : 4340aa3071a0,v0.9.3
+# Source repo   : https://github.com/gonum/gonum
+# Tested on     : ubi 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vathsala . <vaths367@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME="gonum"
+PACKAGE_VERSION=${1:-4340aa3071a0}
+PACKAGE_URL="https://github.com/gonum/gonum"
+OS_NAME=$(grep ^PRETTY_NAME /etc/os-release | cut -d= -f2)
+
+export GO_VERSION=${GO_VERSION:-"1.16"}
+export GOROOT=${GOROOT:-"/usr/local/go"}
+export GOPATH=${GOPATH:-$HOME/go}
+export PATH=$PATH:$GOROOT/bin:$GOPATH/bin:/usr/local/bin
+PACKAGE_SOURCE_ROOT=$(awk -F '/' '{print  "/src/" $3 "/" $4;}' <<<"$PACKAGE_URL" | xargs printf "%s" "$GOPATH")
+export PACKAGE_SOURCE_ROOT
+
+echo "installing dependencies from system repo"
+dnf install -y wget git gcc-c++ make >/dev/null
+
+# installing golang
+wget https://golang.org/dl/go"$GO_VERSION".linux-ppc64le.tar.gz
+tar -C /usr/local/ -xzf go"$GO_VERSION".linux-ppc64le.tar.gz
+rm -f go"$GO_VERSION".linux-ppc64le.tar.gz
+
+if ! git clone "$PACKAGE_URL" "$PACKAGE_SOURCE_ROOT"/"$PACKAGE_NAME"; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    exit 1
+fi
+
+cd "$PACKAGE_SOURCE_ROOT"/"$PACKAGE_NAME"
+git checkout "$PACKAGE_VERSION" || exit 1
+export GO111MODULE=on
+if ! go build -a -v; then
+    echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+#Tests not available(Test N/A).
```

Comment:
@vathsalashetty25 this might have tests, have you looked at the CI? they use go test -v ./... I think